docs/ksm.md

```diff
@@ -0,0 +1,129 @@
+# Kernel Same-Page ([KSM](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/vm/ksm.txt)) tuning
```

Comment:
Let's take the link and abbreviation out of the title and have it in the first instance KSM in regular text. Have the first instance include the KSM in parentheses as above.